pymongoexplain/__init__.py

```diff
@@ -11,3 +11,5 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from pymongoexplain.explainable_collection import ExplainCollection
```

Comment:
There should be a test for this.

Reply:
This is addressed in: https://github.com/mongodb-labs/pymongoexplain/pull/36/commits/50778648da4480a413a65bb47f8f0129bd024e2d